src/js/edition-switch.js

```diff
@@ -4,6 +4,8 @@ const defaultClassName = 'o-header__edition-switch';
 class EditionSwitcher {
 	constructor(headerEl, config = {editionswitchClassName: defaultClassName}) {
 		this.editionSwitcherEl = headerEl.querySelector(`.${config.editionswitchClassName}`);
+		if (!this.editionSwitcherEl) return;
```

Comment:
There's another similar conditional below that returns which is using braces. For consistency I'd say stick to braces, but otherwise no strong opinion either way.